zanstat/Program.cs

```diff
@@ -1,104 +1,153 @@
 ﻿using System;
+using Zanlib;
 
 namespace Zanstat
 {
     class Program
     {
-        static void Main(string[] args)
+        static string hostname = "localhost";
+        static int port = 10666;
+
+        static int Main(string[] args)
         {
-            Zanlib.Huffman.Init();
+            try
+            {
+               
 
-            var hostname = "localhost";
-            var port = 10666;
+                return new Program().QueryZandronumServer(args);
```

Comment:
Again I don't see the value in instantiating here instead of just leaving them as static methods. However, I do think this code needed to be split up into separate functions so thanks for that!